lib/data/model.ts

```diff
@@ -169,6 +169,10 @@ export default class Model extends DenaliObject {
     return new Proxy(this, {
 
       get(model: Model, property: string): any {
+        if (property === 'constructor') {
```

Comment:
Either way I think this is probably a good solution, simply because you get faster returns for things that need to access the constructor (e.g other instance methods)